### ISSUE TITLE
Proper resizing for emojis

### ISSUE THREAD

**Issue by pcroland** [mintty/wsltty#276]
Emoji resizing seems to use an algo that isn't really suited for downscaling.

[!image](https://i.kek.sh/6L6pZ4xTfPE.png)
Left one is from wsltty, right one is resized with ImageMagick (`-resize 15x15`). I tried to resize my 72x72 emojis to 15x15 but then I can't use them wsltty because it reverts back to `none`.

**Comment by mintty** [comment]
Mintty does not scale emoji or other images itself. It uses Windows functions for display so all scaling is performed by Windows.

**Comment by pcroland** [comment]
And would it be possible to allow emojis with custom sizes such as 15x15?

**Comment by mintty** [comment]
Yes, of course. However, the set of supported styles is currently fixed, due to different filename patterns by emoji respositories.
That means you'd have to drop-in the rescaled emojis instead of the original ones, losing access to the higher resolution.

**Comment by pcroland** [comment]
I have them all resized in a different folder and when I set them the settings will revert to `none` as selected emoji pack.

**Comment by mintty** [comment]
> the set of supported styles is currently fixed

This means you cannot invent a new style name.

**Comment by pcroland** [comment]
Oh right, renaming `apple_15px` to `apple` works, thanks.
